lua/entities/gmod_wire_digitalscreen/shared.lua

```diff
@@ -8,6 +8,5 @@ ENT.Purpose         = ""
 ENT.Instructions    = ""
```

Comment:
I guess we should drop these empty `Author`/`Contact`/`Purpose`/`Instructions` too.

Reply:
Separate PR, or I rename this and do it here?